exp-player/addon/components/exp-exit-survey.js

```diff
@@ -123,37 +138,47 @@ export default ExpFrameBaseComponent.extend({
             };
             return {
                 schema: this.get('form.schema'),
-                options: newOptions        
+                options: newOptions
             };
         },
         set(_, value) {
             this.set('formSchema', value);
             return value;
         }
-        
     }),
     section1: true,
     formData: [],
     formActions: Ember.computed(function() {
         var root = this;
         return {
-            update: function () {
+            update: function() {
                 this.refreshValidationState(true);
                 if (this.isValid(true)) {
-                   root.set('formData', this.getValue());
-                   root.set('section1', false);
+                    root.set('formData', this.getValue());
+                    root.set('section1', false);
                 }
             },
             finish: function() {
                 var formData = root.get('formData');
                 Ember.merge(formData, this.getValue());
                 root.set('formData', formData);
-                console.log('Post-study survey complete.')
                 root.actions.next();
             }
-        }
+        };
+    }),
+    currentSessionsCompleted: Ember.computed('frameContext', function() {
+        return (this.get('frameContext.pastSessions') || []).length;
+    }),
+    currentDaysSessionsCompleted: Ember.computed('frameContext', function() {
+        var pastSessionDates = this.get('frameContext.pastSessions').map((session) => {
+            return moment(session.get('createdOn'));
+        });
+        var minDate = moment.min(pastSessionDates);
+        var maxDate = moment.max(pastSessionDates);
+
+        return maxDate.diff(minDate, 'days');
```

Comment:
From discussion with Sam: this will require action (a followup question with the client, perhaps). A TODO or even a JIRA ticket would be recommended to make sure that we don't forget to come back and fix this feature later.

Also, based on which definition we choose, it might help to rename this (or add a comment) to make the purpose and definition more clear.